index.html

```diff
@@ -6,7 +6,7 @@
       name="viewport"
       content="initial-scale=1,maximum-scale=1,user-scalable=no"
     />
-    <title>Active Travel Intervention Platform v1</title>
+    <title>Active Travel Infrastructure Platform v1</title>
```

Comment:
Also, I realized I've typod the name of ATIP in a few places!

Reply:
Woa, I didn't spot that either. Good fix!